src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseException.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+
+namespace NuGet.Packaging.Licenses
+{
+    /// <summary>
+    /// NuGet's internal representation of a license exception identifier. 
+    /// </summary>
+    public class NuGetLicenseException
+    {
+        /// <summary>
+        /// The Exception's identifier
+        /// </summary>
+        public string Identifier { get; }
+
+        private NuGetLicenseException(string identifier)
+        {
+            Identifier = identifier ?? throw new ArgumentNullException(nameof(identifier));
+        }
+
+        /// <summary>
+        /// Parse an exceptionIdentifier. If the exceptionIdentifier is deprecated, this will throw. Non-standard exception do not get parsed into an object model.
+        /// </summary>
+        /// <param name="exceptionIdentifier"></param>
+        /// <returns>NuGetLicenseException</returns>
```

Comment:
Add more detail description to `returns` tag. The return type is already given by the function so no need to have it in the comments.